static/sass/ds/6-components/_messages.scss

```diff
@@ -0,0 +1,9 @@
+.c-messages {
+  > .c-message {
+    margin-bottom: $size-xxl;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
```

Comment:
Same specificity note here. I don't really see a reason for the `>`, and then having to override it again by nesting :last-child within that
```scss
.c-messages {
 .c-message {
   &:last-child {
   }
 }
}
```
seems like it'd still accomplish the same thing

Reply:
Done.